### DISCUSSION TITLE
Inline images

### DISCUSSION THREAD

**Discussion by robinparker1** [x-govuk/govuk-design-history-template#168] [Q&A] [answered]
Hi folks, it's been a while since I used this template, and I'm sure last time I used it I was able to include inline images in posts, rather than listing them at the end. But I can't remember how to do it! Can anyone help?

**Comment by fofr** [comment, marked as answer]
You can use the markdown for images, for example:

```
![Activities associated with a case in TMS](TMS-activities.png "Activities associated with a case in TMS")
```

Where:

```
![alt text](filename.png "Caption text in figure")
```

The design history will automatically look in the same directory you would put screenshots in.

**Reply by robinparker1** [reply]
Thanks!
